cmd/util/flags/configflags/libp2p.go

```diff
@@ -10,11 +10,17 @@ var Libp2pFlags = []Definition{
 		Description: `A comma-separated list of libp2p multiaddress to connect to. ` +
 			`Use "none" to avoid connecting to any peer, ` +
 			`"env" to connect to the default peer list of your active environment (see BACALHAU_ENVIRONMENT env var).`,
+		Deprecated: true,
+		DeprecatedMessage: "The libp2p transport will be deprecated in a future version in favour of using " +
+			"--orchestrators to specify a requester node to connect to.",
```

Comment:
"... Unless you specifically use the features of libp2p (which ones?), you should not have to change anything."

Reply:
I don't think there are any features we are losing due to deprecation of libp2p. In general users will need to change very little, and I'd prefer to describe what they _do_ need to change in a changelog/release-notes/blog-post rather than the CLI help message. 

Perhaps we should create a github discussion to talk about this more and provide a link to the discussion post in the deprecation message for these flags?

Reply:
Yeah we need to start communicating this with users and collect feedback, whether through github discussions or slack channels. cc @seanmtracey